README.md

```diff
@@ -0,0 +1,2 @@
+Javascript reference implementation for
+[Linked Data Capabilities](https://w3c-ccg.github.io/ld-ocap/).
```

Comment:
Totally minor, but this got moved to `https://w3c-ccg.github.io/ocap-ld/`

Reply:
Fix https://github.com/digitalbazaar/ocapld.js/pull/1/commits/7dc80887e8d451621f516622e90cbdacf3fe2120